src/main/java/DenisCamacho/task_02/Table.java

```diff
@@ -0,0 +1,94 @@
+package main.java.DenisCamacho.task_02;
+
+public class Table {
+    private String[][] table;
+
+    public Table() {
+        table = new String[3][3];
```

Comment:
@denis1729 you should use this statement to refer the attributes and methods to the current class